src/content/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more.mdx

```diff
@@ -11,23 +11,23 @@ redirects:
   This doc is for managing users on our [newer user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-user-models). For managing users on our original user model, see [Original users](/docs/accounts/original-accounts-billing/original-users-roles).
 </Callout>
 
-To manage their users, New Relic organizations can configure one or more authentication domains, which control how users are added to a New Relic account, how they’re authenticated, and more.
+To manage their users, New Relic organizations can configure one or more authentication domains, which control how users are added to a New Relic account, how they're authenticated, and more.
 
-## What is an authentication domain? [#auth-domain-definition]
+## Authentication domains explained [#auth-domain-definition]
 
-An "authentication domain" is a grouping of New Relic users governed by the same user management settings, like [how they're provisioned](#source-users) (added and updated), [how they're authenticated](#authentication) (logged in), [session settings](#session-mgmt), and [how user upgrades are managed](#session-mgmt).
+An **authentication domain** is a grouping of New Relic users governed by the same user management settings, like [how they're provisioned](#source-users) (added and updated), [how they're authenticated](#authentication) (logged in), [session settings](#session-mgmt), and [how user upgrades are managed](##user-upgrade).
```

Comment:
Does the final link have 2 #s on purpose?

Reply:
good eye!